
Don't hide all editors in config if BR2_PACKAGE_BUSYBOX_HIDE_OTHERS is selected.
Indeed vi is the only editor already provided by busybox.

Signed-off-by: Julien Boibessot <julien.boibessot@armadeus.com>

Integrated in BR for 2010.02.

---

Index: buildroot/package/Config.in
===================================================================
--- buildroot.orig/package/Config.in	2010-01-25 10:09:17.000000000 +0100
+++ buildroot/package/Config.in	2010-01-25 10:09:29.000000000 +0100
@@ -125,14 +125,9 @@
 
 source "package/database/Config.in"
 
-if !BR2_PACKAGE_BUSYBOX_HIDE_OTHERS
-# busybox has an editor
-
 # text editors
 source "package/editors/Config.in"
 
-endif
-
 menuconfig BR2_NETWORK_SUPPORT
 	bool "Networking"
 	default y
Index: buildroot/package/editors/Config.in
===================================================================
--- buildroot.orig/package/editors/Config.in	2010-01-25 10:02:52.000000000 +0100
+++ buildroot/package/editors/Config.in	2010-01-25 10:09:29.000000000 +0100
@@ -8,5 +8,8 @@
 source "package/editors/ed/Config.in"
 source "package/editors/nano/Config.in"
 source "package/editors/uemacs/Config.in"
+if !BR2_PACKAGE_BUSYBOX_HIDE_OTHERS
+# busybox has vi
 source "package/editors/vim/Config.in"
 endif
+endif
